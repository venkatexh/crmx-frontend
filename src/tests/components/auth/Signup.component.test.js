import { shallow } from "enzyme";
import React from "react";
import SignupComponent from "../../../components/auth/Signup.component";

it("Signup component snapshot", () => {
  expect(shallow(<SignupComponent />)).toMatchSnapshot();
});
