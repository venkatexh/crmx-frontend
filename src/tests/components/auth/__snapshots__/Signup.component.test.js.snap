// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup component snapshot 1`] = `ShallowWrapper {}`;
